BetterMD/elements/table.py

```
@@ -3,91 +3,157 @@
 from ..rst import CustomRst
 from .h import H1, H2, H3, H4, H5, H6
 from .text import Text
+import itertools as it
```

Ruff (0.8.2)

`itertools` imported but unused

Remove unused import: `itertools`

(F401)